components/automate-chef-io/content/docs/upgrade.md

```diff
@@ -180,6 +180,6 @@ At this point, you can use your existing Chef Automate 1 user credentials to log
 If you've been using LDAP for authenticating users, that configuration will have been migrated as well, and you can use your LDAP credentials to login.
 Historical data will be migrated in the background.
 
-### Upgrading an Airgapped Installation
+### Upgrading an airgapped installation between versions of Chef Automate 2
```

Comment:
i'm not sure this new title is necessary

Reply:
@jaym I'll take that as permission to do what I wanted to do in the first place...